cmd/vulcan-exposed-ssh/main.go

```diff
@@ -176,10 +184,12 @@ func (r *runner) gradeVuln() ([]report.Vulnerability, error) {
 			// NOTE: this doesn't cover different types of errors that ssh_scan can produce
 			continue
 		}
-		vulnDb := map[string]report.Vulnerability{}
 		// Exposed SSH
-		exposedSSHVuln.Details += fmt.Sprintf("* Exposed SSH Port in %v\n", i.Port)
-		vulnDb["exposedSSHVuln"] = exposedSSHVuln
+		v := exposedSSHVuln
+		v.Details += fmt.Sprintf("* Exposed SSH Port in %v\n", i.Port)
+		v.AffectedResource = fmt.Sprintf("%d/%s", i.Port, "ssh")
```

Comment:
I'd rather use something like `22/tcp` instead of `ssh`.

Reply:
Done!